### ISSUE TITLE
Yum lens saving excludes incorrectly

### ISSUE THREAD

**Issue by cfenoy** [hercules-team/augeas#487]
When using the Yum lens, it is able to parse properly exclude lists, but it is unable to add a new item to the list properly.

$ cat /etc/yum.repos.d/epel.repo
[epel]
enabled=0
exclude=kernel* kernel2*

$ augtool
augtool> set /files/etc/yum.repos.d/epel.repo/epel/exclude[last()+1] "kernel3*"
augtool> print /files/etc/yum.repos.d/epel.repo/
/files/etc/yum.repos.d/epel.repo
/files/etc/yum.repos.d/epel.repo/epel
/files/etc/yum.repos.d/epel.repo/epel/enabled = "0"
/files/etc/yum.repos.d/epel.repo/epel/exclude[1] = "kernel*"
/files/etc/yum.repos.d/epel.repo/epel/exclude[2] = "kernel2*"
/files/etc/yum.repos.d/epel.repo/epel/exclude[3] = "kernel3*"
augtool> save
Saved 1 file(s)
augtool>
[root@localhost tmp]# cat /etc/yum.repos.d/epel.repo
[epel]
enabled=0
exclude=kernel* kernel2*
	kernel3*

**Comment by lutter** [comment]
I just checked this out - and what the lens does is indeed correct, even if it looks a bit goofy. Yum allows that list entries are continued on the next line as long as that line starts with whitespace. The lens adds a tab at the beginning of the `kernel3*` line, and as far as I can tell, that's all legal.

We could change the cosmetics of this to make exclude add new entries by just putting a space between them, but I am not sure if that's worth the effort.

BTW, I used this script to experiment with how yum actually parses its config files:
```python
import yum

my=yum.YumBase()
[epel]=[ r for r in my.repos.listEnabled() if r.id=='epel']

print epel.id
print epel.exclude
print epel.baseurl
print epel.gpgkey
```
